packages/core/src/Stores/client-store.js

```diff
@@ -2263,12 +2264,15 @@ export default class ClientStore extends BaseStore {
             this.setIsLoggingIn(true);
 
             const redirect_url = sessionStorage.getItem('redirect_url');
+            const is_next_wallet_enabled = localStorage.getObject('FeatureFlagsStore')?.data?.next_wallet;
+
+            const target_url = is_next_wallet_enabled ? routes.wallets : routes.traders_hub;
 
             if (
                 (redirect_url?.endsWith('/') || redirect_url?.endsWith(routes.bot)) &&
-                (isTestLink() || isProduction() || isLocal() || isStaging())
+                (isTestLink() || isProduction() || isLocal() || isStaging() || isTestDerivApp())
```

Comment:
wondering about uat, 
we have now:
test-app.deriv.com
staging-app.deriv.com
uat-app.deriv.com
app.deriv.com 
:D

Reply:
None of us are using uat environment for development or testing as far as i know. The rest are covered 😸